lib/competition_results_import.rb

```diff
@@ -35,4 +35,42 @@ def self.import_temporary_results(competition, temporary_results_data, mark_resu
 
     errors
   end
+
+  def self.merge_temporary_results(ticket)
```

Comment:
This method can only get `competition` as a parameter. If you need to do ticket-related changes, you should do them outside of this method, at the place where the method is called.

Reply:
Actually I also need to update the status of the ticket. This is called from two places and I need to do the same thing in both places. So is it fine to pass ticket instead like now?

Reply:
Okay I've done it with help of ticket_competition_result model. Hope this is fine now?